README.md

```diff
@@ -688,6 +688,12 @@ However this situation already exists.
 Today, many users omit a build step and ship large amounts of comments and other extraneous information, e.g. unminified code.
 It remains a best practice to perform an ahead-of-time optimization step on code destined for production if the use-case is performance-sensitive.
 
+Specifically, note that [the TypeScript compiler `tsc` does not have a built-in option to minify](https://github.com/microsoft/TypeScript/issues/8).  Allowing TypeScript users to skip `tsc` during development will not inherently encourage users to skip a minification step, since that would have required separate action in any case.
```

Comment:
```suggestion
Specifically, note that [the TypeScript compiler `tsc` does not have a built-in option to minify](https://github.com/microsoft/TypeScript/issues/8). Allowing TypeScript users to skip `tsc` during development will not inherently encourage users to skip a minification step, since that would have required separate action in almost any case.
```

Reply:
Why "almost"? Not every developer will minify, but, if minification would be done, it would necessarily require separate action, right?

Reply:
Because of the Closure Compiler comment above.

Reply:
Ah, I see the confusion. When I said "that" would have required separate action, I meant "minifying TypeScript™" not "minification in general." I've rewritten the sentence to say:

> Allowing TypeScript users to skip `tsc` during development will not inherently encourage users to skip a minification step, since minifying TypeScript would have required a separate action in any case.

(Note that Closure Compiler uses a JSDoc syntax for types, and doesn't transpile in-line type parameters as TypeScript and Flow do. You can't use CC to transpile TypeScript.)